packages/web-components/src/components/va-search-input/va-search-input.scss

```diff
@@ -1,7 +1,49 @@
-@import '../../mixins/buttons.css';
-@import '../../mixins/inputs.css';
+@forward 'settings';
+
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-search/src/styles/usa-search';
+@use 'usa-button/src/styles/usa-button';
+
+@import '../../mixins/uswds-error-border.scss';
+@import '../../global/formation_overrides';
 
 :host {
+  display: block;
+  border: none;
+  padding: 0;
+}
+
+:host([uswds]:not([uswds='false'])) .usa-input {
+  margin-top: 0;
+}
+
+:host([uswds]:not([uswds='false'])) #va-search-listbox {
+  top: 37px;
+  z-index: 1;
+}
+
+:host([uswds]:not([uswds='false'])[big="true"]) #va-search-listbox  {
+  top: 52px;
+}
+
+.usa-search [type=submit] {
+  width: auto;
+}
+
+.usa-button {
+  line-height: 0;
```

Comment:
Because I am using `va-icon` instead of an `img` element, I needed to add this line-height reset otherwise the icon was sitting a pixel or two off center.